ports/esp32/boards/LOLIN_S2_PICO/modules/s2pico.py

```diff
@@ -0,0 +1,104 @@
+# LOLIN S2 PICO MicroPython Helper Library
+
+
+from micropython import const
+from machine import Pin, SPI, ADC, I2C
+import machine, time
+import ssd1306
+
+import network
+
+
+# Pin Assignments
+
+# SPI
+SPI_MOSI = const(35)
+SPI_MISO = const(36)
+SPI_CLK = const(37)
+
+# I2C
+I2C_SDA = const(8)
+I2C_SCL = const(9)
+
+# DAC
+DAC1 = const(17)
+DAC2 = const(18)
+
+# LED
+LED = const(10)
+
+# OLED
+OLED_RST = const(18)
+
+# BUTTON
+BUTTON = const(0)
+
+
+class led:
+    def __init__(self):
+        self._led = Pin(LED, Pin.OUT, value=1)
+
+    def on(self):
+        self._led.value(0)
+
+    def off(self):
+        self._led.value(1)
+
+
+class button:
+    def __init__(self):
+        self._btn = Pin(BUTTON, Pin.IN, Pin.PULL_UP)
+
+    def get_button_status(self):
+        return self._btn.value()
+
+    def set_button_callback(self, cb):
+        self._btn.irq(trigger=Pin.IRQ_FALLING, handler=cb)
+
+
+class oled(ssd1306.SSD1306_I2C):
+    def __init__(self):
+        self.reset()
+
+        # super().__init__(128,32,I2C(0))
```

Comment:
This line should probably be uncommented (__init__ should usually only be called from within __init__).